pkg/proc/eval.go

```diff
@@ -288,6 +287,10 @@ func (scope *EvalScope) extractVarInfoFromEntry(varEntry *dwarf.Entry) (*Variabl
 	return v, nil
 }
 
+func (scope *EvalScope) extractVarInfoFromEntry(varEntry *dwarf.Entry) (*Variable, error) {
```

Comment:
Do we still need this at all?

Reply:
I thought it was used in multiple places but it's actually only called in one place so I removed it now.